### PR TITLE
Bring back advanced pagination buttons

### LINKED ISSUE #4175: Advanced pagination is missing the buttons
In Production 2, buttons for entering special characters are offered for advanced programming. These also serve as a memory aid for the operator.

Buttons in advanced pagination

These buttons are missing in Production 3.

**Goal:** add the missing buttons.

---

**Text:** Without selection, inserts two back-ticks (<code>``</code>) in position of the cursor and places the cursor between them. With selection, surrounds the selection with back-ticks.

**+0:** Inserts a superscript zero (degree character, `°`)

**+½:** Inserts a vulgar one-half fraction character (`½`)

**+1:** Inserts a superscript one (`¹`)

**+2:** Inserts a superscript two (`²`)

**+3:** Inserts a superscript three (`³`)

**[1]:** Inserts an upside-down exclamation sign (`¡`)

**[½]:** Inserts an upside-down question mark (`¿`)

For an explanation of the functionality behind this, see [Paginieren](https://github.com/kitodo/kitodo-production/wiki/Paginieren#paginierung-f%C3%BCr-fortgeschrittene).

### LINKED ISSUE #4475: Cannot enter text in free text pagination
In free text pagination, no text can be entered, only numbers.

Screenshot

**Goal:** In free text pagination, it should be possible to enter anything.

### PR DESCRIPTION
**Pull request by matthias-ronge**
Fixes #4175, fixes #4475

### CODE REVIEW

**Review by Kathrin-Huber — CHANGES_REQUESTED** [review]
Please put this branch on the preview system and let it check from a product owner

File: Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/PaginationPanel.java [hunk 1]
```diff
@@ -407,4 +404,61 @@ public void show() {
...
+     * Is text.
+     */
+    public void isText() {
+        paginationStartValue = '`' + paginationStartValue + '`';
+    }
+
+    /**
+     * No increment.
+     */
+    public void noIncrement() {
```
[the remark this answers is not in the document]
Comment (matthias-ronge) [reply Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/PaginationPanel.java:418]: See section [Paginierung für Fortgeschrittene](https://github.com/kitodo/kitodo-production/wiki/Paginieren#paginierung-f%C3%BCr-fortgeschrittene) in the wiki

**Comment by matthias-ronge** [comment]
> Please put this branch on the preview system and let it check from a product owner

I don’t know how the test system is meant to be.

**Comment by solth** [comment]
> > Please put this branch on the preview system and let it check from a product owner
> 
> I don’t know how the test system is meant to be.

@matthias-ronge @Kathrin-Huber I just merged this branch into the preview system. The changes can now be tested there.

**Comment by Kathrin-Huber** [comment]
paginierung

The buttons always add-on, is this expected behaviour?
If I want to change the input, it restores with the next button. (see gif)

**Comment by matthias-ronge** [comment]
> The buttons always add-on, is this expected behaviour?

Yes.

> If I want to change the input, it restores with the next button. (see gif)

That shouldn't happen now any more. And, now it works on the currently highlighted selection (which is intentional):

Animation

**Pull request merged**